docs/source/conf.py

```diff
@@ -145,13 +145,8 @@ class DitaaLexer(BashLexer):
 #
 html_theme_options = {
     'branch_substring_removed': 'scylla-',
-    'header_links': [
-    ('Scylla Java Driver', 'https://java-driver.docs.scylladb.com/'),
-    ('Scylla Cloud', 'https://docs.scylladb.com/scylla-cloud/'),
-    ('Scylla University', 'https://university.scylladb.com/'),
-    ('ScyllaDB Home', 'https://www.scylladb.com/')],
     'github_issues_repository': 'scylladb/java-driver',
-    'show_sidebar_index': True,
+    'hide_sidebar_index': 'False',
```

Comment:
```suggestion
    'hide_sidebar_index': 'false',
```

Should work as is too, but best if we put it in lowercase as we have done in other projects.